backend/typings/ceph_module.pyi

```diff
@@ -3,113 +3,140 @@
 # Without this file, all classes have undefined base classes.
 
 from typing import Any, Dict, List, Mapping, Optional, Sequence, Tuple, Union
+
 try:
     from typing import Protocol  # Protocol was added in Python 3.8
 except ImportError:
     class Protocol:  # type: ignore
         pass
 
-
 class BasePyOSDMap(object):
     def _get_epoch(self): ...
     def _get_crush_version(self): ...
-    def _dump(self):...
-    def _new_incremental(self):...
-    def _apply_incremental(self, inc: 'BasePyOSDMapIncremental'):...
-    def _get_crush(self):...
-    def _get_pools_by_take(self, take):...
-    def _calc_pg_upmaps(self, inc, max_deviation, max_iterations, pool):...
-    def _map_pool_pgs_up(self, poolid):...
-    def _pg_to_up_acting_osds(self, pool_id, ps):...
-    def _pool_raw_used_rate(self, pool_id):...
+    def _dump(self): ...
+    def _new_incremental(self): ...
+    def _apply_incremental(self, inc: "BasePyOSDMapIncremental"): ...
+    def _get_crush(self): ...
+    def _get_pools_by_take(self, take): ...
+    def _calc_pg_upmaps(self, inc, max_deviation, max_iterations, pool): ...
+    def _map_pool_pgs_up(self, poolid): ...
+    def _pg_to_up_acting_osds(self, pool_id, ps): ...
+    def _pool_raw_used_rate(self, pool_id): ...
```

Comment:
I think this file was generated using `stubgen` ...  maybe we should ignore the type errors in this file?

Reply:
Ah, fair enough, I agree we should keep ignoring these then.